ui.frontend/src/main/webpack/components/_breadcrumb.scss

```diff
@@ -1,5 +1,20 @@
-.cmp-breadcrumb {}
-.cmp-breadcrumb__list {}
-.cmp-breadcrumb__item {}
-.cmp-breadcrumb__item--active {}
-.cmp-breadcrumb__item-link {}
\ No newline at end of file
+.cmp-breadcrumb__list {
```

Comment:
please update styles. Current page item shouldn't be bold

Reply:
<img width="1246" alt="Screenshot 2024-05-30 at 9 22 09 AM" src="https://github.com/lflondonol92/prft-aem-bootcamp-latam-q2/assets/25993793/d1c003f6-d2d9-446c-aa32-fb456937efd5">

New  Commit will fix it